pkg/operator/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -239,9 +239,75 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Monitor the cloud-credential-operator-config configmap to check if operator is re-enabled.
+	// Check if attempts were made to delete the credentials requests when the operator was disabled.
+	// We have to have to reconcile all those credential requests.
+	configmapMapFn := handler.ToRequestsFunc(
+		func(a handler.MapObject) []reconcile.Request {
+			// Iterate all CredentialsRequests to determine if any of them have DeletionTimestamp set
+			log.Debug("checking for credentials requests marked for cleanup when operator was disabled")
```

Comment:
Let's make this simpler, just requeue them all, there's no need to only requeue the deleted ones.